src/kokkos_abstraction.hpp

```diff
@@ -785,6 +785,16 @@ std::unique_ptr<T, DeviceDeleter<MS>> DeviceCopy(const T &host_object) {
   return up;
 }
 
+template <typename T, typename ES = DevExecSpace, typename MS = DevMemSpace>
+T *RawDeviceCopy(const T &host_object) {
+  auto p = static_cast<T *>(Kokkos::kokkos_malloc<MS>(sizeof(T)));
+  Kokkos::parallel_for(
+      Kokkos::RangePolicy<ES>(0, 1),
+      KOKKOS_LAMBDA(const int i) { new (p) T(host_object); });
+  Kokkos::fence();
+  return p;
+}
+
```

Comment:
Is this called somewhere?

Reply:
No but I figured it might be useful in downstream codes (at one point I found myself in a spot where getting a `unique_ptr` back was frustrating since there it was returned as a temporary object). I don't think I'm using that anymore if you'd rather remove this.

Reply:
If you already use it downstream, then we should keep it (and somehow test it).
If it's not used I'm in favor if removing it so that it can be introduced/tested when required.

Reply:
You're right, this shouldn't be floating around untested. I'm not using this now so I've removed it.